### PR TITLE
Moving setICUDataDirectory inside zim namespace

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Moving setICUDataDirectory inside zim namespace

### CODE REVIEW

File: src/tools.h [hunk 1]
```diff
@@ -55,7 +55,7 @@ namespace zim {
                                 const std::string & delims=" *-");
 
   std::map<std::string, int> read_valuesmap(const std::string& s);
-
+  void setICUDataDirectory(const std::string& path);
```
Comment (mgautierfr) [inline src/tools.h:58]: setICUDataDirectory is already (publicly) declared in (`zim/tools.h`)[https://github.com/openzim/libzim/blob/master/include/zim/tools.h#L35].

So we don't need to declare it here.

**Review by mgautierfr — CHANGES_REQUESTED** [review]
We only need the change in src/tools.cpp.

And it should be only one commit base on master. (`git rebase` is your friend here)

File: src/tools.h (hunk 1 above)
Comment (MohitMaliFtechiz) [reply src/tools.h]: done

**Comment by kelson42** [comment]
@mgautierfr Looks good now?

**Pull request merged**